re.c

```diff
@@ -71,6 +71,7 @@ static int ismetachar(char c);
 /* Public functions: */
 int re_match(const char* pattern, const char* text, int* matchlength)
 {
+#if defined(RE_ENABLE_MULTI_PATTERNS) && (RE_ENABLE_MULTI_PATTERNS == 1)
```

Comment:
I'm not sure about this part but in my understanding one don't have to check ``defined(RE_ENABLE_MULTI_PATTERNS)`` every time, just a single check in ``re.h`` (``#ifndef ... #define ... #endif``) seems to be enough and it's already there. At the same time I see that this construction is common for both this and other repos of the author, so your change is consistent with the other code. It's interesting what @kokke thinks about such checks.

Reply:
emmmm , fix it